tests/jobs/scala/src/main/scala/KafkaJobs.scala

```diff
@@ -0,0 +1,162 @@
+import java.util
+
+import org.apache.spark.{SparkConf, SparkContext}
+import org.apache.kafka.clients.producer.{KafkaProducer, ProducerConfig, ProducerRecord}
```

Comment:
[warn] /home/ubuntu/repos/OSS/spark-build/tests/jobs/scala/src/main/scala/KafkaJobs.scala:4: imported `KafkaProducer' is permanently hidden by definition of object KafkaProducer

Reply:
Also please use a package for the jobs in that folder like: dco.spark.test